sam/build-application/action.yml

```diff
@@ -71,7 +74,7 @@ runs:
       env:
         OS: ${{ runner.os }}
         CACHE_NAME: ${{ inputs.cache-name }}
-        HASH: ${{ hashFiles(inputs.template || '**/template.y*ml', inputs.source-dir) }}
+        HASH: ${{ hashFiles(inputs.template || '**/template.y*ml', inputs.source-dir, inputs.manifest) }}
```

Comment:
How confident are we that when `manifest` is not provided that this results in the same hash result as the original list of inputs? Do we have a test for this?

Reply:
Sorry @kalpaitch, my question could've been phrased more clearly 😓 Before/after this change is merged, will an existing consumer of the action see the same hash for a given set of input files, assuming they don't populate the `manifest`? I'm trying to work out whether the build artifacts are still effectively hermetic regardless of which version of the action a team is using to generate the hash.